pcli/src/command/view/balance.rs

```diff
@@ -18,8 +18,8 @@ pub struct BalanceCmd {
 }
 
 impl BalanceCmd {
-    pub fn needs_sync(&self) -> bool {
-        !self.offline
+    pub fn offline(&self) -> bool {
+        self.offline
```

Comment:
This doesn't work any more, because now we're combining "does the command require a view service" with "does the command work offline". Now, the offline option for balance won't work, because the view service won't be present.

To fix this, we can remove the option entirely, and have the balance command always require a sync.